cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
-  note: This VCC was discovered automatically via archeogit.
-- commit: fcd837cd0f9b2c706bc49af509628778d442bb3f
-  note: This VCC was discovered automatically via archeogit.
-- commit: afd040d4d3a06fe92e3080870b2ff2095ce86a75
-  note: This VCC was discovered automatically via archeogit.
-- commit: 19b9211a3b5424e7908a288c5008bf972cc472f4
-  note: This VCC was discovered automatically via archeogit.
-- commit: e74edb4d53b089ec57ec4830eeba98607283a092
-  note: This VCC was discovered automatically via archeogit.
-- commit: fc6649092aae5667ee6ed9b16ed81ebbe41a7bf4
-  note: This VCC was discovered automatically via archeogit.
-- commit: 4b610f42d36a39cd975122ab46f85323844c2c72
-  note: This VCC was discovered automatically via archeogit.
-- commit: 7a60b411305bc363ec51a096b108fa909c1b352b
-  note: This VCC was discovered automatically via archeogit.
-- commit: 5df31c0164e9477a3ebb6b1bbde8604e06fbefd4
-  note: This VCC was discovered automatically via archeogit.
-upvotes_instructions: |
-  For the first round, ignore this upvotes number.
-
-  For the second round of reviewing, you will be giving a certain amount of
-  upvotes to each vulnerability you see. Your peers will tell you how
-  interesting they think this vulnerability is, and you'll add that to the
-  upvotes score on your branch.
-upvotes:
-unit_tested:
+curated: true
+curated_instructions: |
+    If you are manually editing this file, then you are "curating" it. Set the
+    entry below to "true" as soon as you start. This will enable additional
+    integrity checks on this file to make sure you fill everything out properly.
+    If you are a student, we cannot accept your work as finished unless curated is
+    set to true.
+description: "The function get_host could be exploited to generate arbitrary URLs. An attacker could put malicious username and password values in the Host header that enabled this."
+description_instructions: |
+    You can get an initial description from the CVE entry on cve.mitre.org. These
+    descriptions are a fine start, but they can be kind of jargony.
+    
+    Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
+    read to anyone with some programming experience. We can always pull up the NVD
+    description later to get more technical.
+    
+    Try to still be specific in your description, but remove project-specific
+    stuff. Remove references to versions, specific filenames, and other jargon
+    that outsiders to this project would not understand. Technology like "regular
+    expressions" is fine, and security phrases like "invalid write" are fine to
+    keep too.
+    
+    Your target audience is people just like you before you took any course in
+    security
+discovered: 
+  answer: "The vulnerability was discovered by James Kettle. There is no explanation for how it was found. https://www.djangoproject.com/weblog/2012/oct/17/security/\n"
+  automated: false
+  contest: false
+  developer: true
```

Comment:
Is James Kettle a Django developer?  It looks like he may just be a security researcher.